src/sdk/src/signed_transaction.rs

```diff
@@ -9,3 +11,11 @@ pub struct SignedTransaction {
     pub header: ::std::collections::HashMap<std::string::String, std::string::String>,
     pub signature: std::vec::Vec<u8>,
 }
+
+impl Message for SignedTransaction {
```

Comment:
This implementation for signedTransaction, can it be moved outside SDK

Reply:
For that we need to move Message Trait also in "message_handler" module.